### PR TITLE
Upgrade Nextra

### PR DESCRIPTION
**Pull request by shuding**
Upgrade Nextra to latest beta that fixes a couple of bugs. This will likely include all the break changes we planned for v2.

Major changes are `meta.json` being renamed to `_meta.json`, and configuration renames. Still working on the docs for all these but we also have full TS covered now, so this can be easier.

### CODE REVIEW

File: docs/components/Tabs.tsx [hunk 1]
```diff
@@ -1,9 +1,14 @@
-import * as NextraTabs from "nextra-theme-docs/tabs";
+import type { FC } from "react";
+
+import { Tabs as NextraTabs, Tab } from "nextra-theme-docs";
```
Comment (shuding) [inline docs/components/Tabs.tsx:3]: Will have to be imported from the theme directly.

File: docs/nextra.d.ts [hunk 2]
```diff
@@ -1 +0,0 @@
-declare module "nextra-theme-docs/tabs";
```
Comment (shuding) [inline docs/nextra.d.ts:1]: No longer needed, the theme is now strongly typed.

File: docs/pages/_app.tsx [hunk 3]
```diff
@@ -1,5 +1,4 @@
 import "../styles.css";
-import "nextra-theme-docs/style.css";
```
Comment (shuding) [inline docs/pages/_app.tsx:2]: Theme CSS and layout are no longer required to be handled in _app.

**Review by mehulkar — APPROVED** [review]
Diff looks good and clicked around a few pages in the preview and that looks good also!

**Pull request merged**